index.js

```diff
@@ -43,7 +44,7 @@ class Aperture {
     videoCodec = undefined
   } = {}) {
     this.processId = getRandomId();
-    return new Promise((resolve, reject) => {
+    return new Promise(async (resolve, reject) => {
```

Comment:
It's an anti-pattern to use `async` in `new Promise` as if the async function rejects, it causes an unhandled rejection. It's better then to use an async-IIFE and try/catch inside it.